etna/metrics/base.py

```diff
@@ -27,6 +28,23 @@ def _missing_(cls, value):
         )
 
 
+class MetricFunctionSignature(str, Enum):
+    """Enum for different metric function signatures."""
+
+    #: function should expect arrays or y_pred and y_true with length ``n_timestamps`` and return scalar
```

Comment:
We should write about this notation in the contribution guide

Reply:
or -> of

Reply:
What about contribution guide?

Reply:
I don't think it is necessary to add this into contribution guide for now. We'll see is it necessary during reviews.